src/lib/CrsfProtocol/crsf_protocol.h

```diff
@@ -301,6 +309,51 @@ typedef struct crsf_sensor_baro_vario_s
     int16_t verticalspd;  // Vertical speed in cm/s, BigEndian
 } PACKED crsf_sensor_baro_vario_t;
 
+// CRSF_FRAMETYPE_AIRSPEED
+typedef struct crsf_sensor_airspeed_s
+{
+    uint16_t speed;             // Airspeed in 0.1 * km/h (hectometers/h)
+} PACKED crsf_sensor_airspeed_t;
+
+// CRSF_FRAMETYPE_RPM
+typedef struct crsf_sensor_rpm_s
+{
+    uint8_t rpm_source_id;      // Identifies the source of the RPM data (e.g., 0 = Motor 1, 1 = Motor 2, etc.)
```

Comment:
I think just source_id is fine rather than duplicating the name of packet type in the structure? Same thing with `temp_source_id` and `Cell_sensor_source_id`

Reply:
Done

Reply:
Done